lib/loadGltfUris.js

```diff
@@ -94,7 +98,12 @@ function loadURI(gltf, basePath, name, callback) {
     }
 }
 
+//Return the extension of the data uri
  function getImageDataUriFormat(uri) {
-    //Return the extension of the data uri
-    return '.' + uri.match('\/([^)]+)\;')[1];
+    var extension = uri.match('\/([^)]+)\;');
+    if (extension === null) {
```

Comment:
This was mainly to address images in binary glTF, where we have the image data but the data URI is just "data:,", so I just used a placeholder since no format is specified (I think? I couldn't find the image format being represented in the binary image object.)